components/scrubber/scrubber.go

```diff
@@ -48,6 +49,16 @@ type TrustedValue interface {
 	IsTrustedValue()
 }
 
+type TrustedValueWrap struct {
```

Comment:
Why TrustedValue does not fit here? Could you add docs why we need this type and when to use which?

Reply:
Or can we change TrustedValue instead?

Reply:
yes, I mean it is rather a library it would be good to have a short comment there for others.